plugins/pdr_deterministic_plugin/ufm_sim_web_service/isolation_mgr.py

```diff
@@ -518,7 +518,43 @@ def read_next_set_of_high_ber_or_pdr_ports(self):
                     issues[port_name] = link_downed_issue
                 elif ber_issue:
                     issues[port_name] = ber_issue
-            return issues
+                # If out of operating conditions we'll overwrite the cause
+                if self.temp_check and self.is_out_of_operating_conf(port_name):
+                    issues[port_name] = Constants.ISSUE_OONOC
```

Comment:
better renaming "issues" to "isolation_candidates"

Reply:
This is an old code that just moved. It is called "issues" in all other places. Let's change it later as part of another PR (one of PDR resilience tasks)

Reply:
ok just add to the rm board